en/gps_compass/rtk_gps_fem_mini2.md

```diff
@@ -0,0 +1,61 @@
+# Femtones MINI2 receiver
+
+[MINI2 Receiver](http://www.femtomes.com) supports the absolute accuracy positioning of Global Navigation Satellite System (GNSS).
+This delivers the quickest and most reliable RTK initializations for centimeter level positioning.
+Mainly facing to applications of high-precision positioning, navigation and mapping, etc. 
+Customers benefit from the Ethernet connectivity available on the board, allowing high speed data transfer and configuration viastandard web browsers.
+Serial port, USB and CAN are also supported.
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_Receiver.jpg" width="500px" title="mini2" />
+
+## Required Firmware Options
+
+The following firmware options need to be selected when buying the device:
+
+- 5Hz 10Hz 20Hz 
+- INS 
+- HEADING
+- OBS
+- RTK
+- BASE 
+
+## Purchase
+
+* [http://www.femtomes.com](http://www.femtomes.com)

```

Comment:
Would I be correct that all purchase are actually by contacting sales? I.e. there is  no online "shop"?

Reply:
yes, we have no online "shop"